content/updates/2020-09/1-Display-labels-in-your-product-exports.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Display labels instead of codes in your product exports"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Display labels instead of codes in your product exports
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/exports.html#update-an-export-profile"
```

Comment:
This link "link-to-doc="../articles/exports.html#update-an-export-profile" does not explain how to configure the export builder to export labels and not codes. Did you write a specific help center page? If not, we will need one. 
We need to guide users on how to configure and use this feature.

Reply:
In this article, there is a specific table where I added the details of each field in the export builder. Unfortunately there is no anchor to link directly to the table.

Reply:
It will be better to add a section "Configure your export profile" before "Update an export profile", and remove the table and explain clearly the options. We miss a screenshot too, and options are not in the right order in this table, Export files and images and Number of lines per file options should appear before your new options


don't hesitate to change the doc if you feel it needs to be improved.

Reply:
Seen with you, we need at least a screenshot of the settings page and the options in the right order.

Reply:
> Seen with you, we need at least a screenshot of the settings page and the options in the right order.

Done!